### ISSUE TITLE
Bug report - MaxMind GeoIp2 Precision doesn't work

### ISSUE THREAD

**Issue by StreamEnviro** [mautic/mautic#1404]
I have a paid subscription to the MaxMind precision service but Mautic does not pick up any of the other information that should be available.

The credentials are placed in Mautic in the correct format username:password.

I am happy to share the credentials with a developer to help sort this issue out.

**Comment by dongilbert** [comment]
What other information are you wanting it to pick up? Can you give an example of the data that is available to a MaxMind Precision user that would be useful to include?

**Comment by StreamEnviro** [comment]
The most important items are the company name and address but this is a list of the data it gives when you use the service directly on the MaxMind website.

{"subdivisions":[{"iso_code":"ENG","geoname_id":6269131,"confidence":23,"names":{"en":"England","de":"England","fr":"Angleterre","zh-CN":"英格兰","es":"Inglaterra","ru":"Англия","ja":"イングランド","pt-BR":"Inglaterra"}}],"continent":{"geoname_id":6255148,"code":"EU","names":{"en":"Europe","de":"Europa","fr":"Europe","zh-CN":"欧洲","es":"Europa","ja":"ヨーロッパ","ru":"Европа","pt-BR":"Europa"}},"traits":{"organization":"Orange mobile","ip_address":"213.205.198.95","isp":"EE Mobile","user_type":"cellular","autonomous_system_organization":"EE Limited","autonomous_system_number":12576},"maxmind":{"queries_remaining":10998},"postal":{"code":"EC4N"},"location":{"latitude":51.5144,"longitude":-0.0941,"time_zone":"Europe/London","accuracy_radius":47},"registered_country":{"names":{"pt-BR":"Reino Unido","es":"Reino Unido","ru":"Великобритания","ja":"イギリス","zh-CN":"英国","fr":"Royaume-Uni","de":"Vereinigtes Königreich","en":"United Kingdom"},"geoname_id":2635167,"iso_code":"GB"},"country":{"names":{"zh-CN":"英国","pt-BR":"Reino Unido","ja":"イギリス","ru":"Великобритания","es":"Reino Unido","en":"United Kingdom","fr":"Royaume-Uni","de":"Vereinigtes Königreich"},"confidence":99,"geoname_id":2635167,"iso_code":"GB"},"city":{"confidence":13,"names":{"zh-CN":"伦敦","pt-BR":"Londres","ru":"Лондон","ja":"ロンドン","es":"Londres","en":"London","fr":"Londres","de":"London"},"geoname_id":2643743}}

**Comment by StreamEnviro** [comment]
See https://www.mautic.org/community/index.php/1103-maxmind-geoip2-not-working/p1#p5355 for solution.

**Comment by mellsaid** [comment]
Hi All, 

We have a maxmind precision service. The maxmind precision service seems to be not working. We tried to use the solution suggested but we think the API may have changed from maxmind that's why authentication is not working? We also tried to choose maxmind insights and precision options which our suscription allows us to access - but none of them return results when setup on mautic. 

We tried username:pass and tried the authentication key provided by maxmind to call the APIs.
